tests/monotouch-test/ObjCRuntime/RegistrarTest.cs

```diff
@@ -1066,6 +1073,13 @@ class StaticDerivedClass : StaticBaseClass
 		{
 		}
 		
+		[Register ("InvalidDateTime")]
+		class InvalidDateTime : NSObject
+		{
+			[Export ("foo:")]
+			public void Foo (DateTime t) { }
+		}
```

Comment:
This is [failing at build time](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/13952/Test_20Report/) with:
```
15:58:03.7762920 /Users/builder/jenkins/workspace/xamarin-macios-pr-builder/tests/monotouch-test/ObjCRuntime/RegistrarTest.cs(1080): error MT4102: The registrar found an invalid type `System.DateTime` in signature for method `MonoTouchFixtures.ObjCRuntime.RegistrarTest/InvalidDateTime.Foo`. Use `Foundation.NSDate` instead. [/Users/builder/jenkins/workspace/xamarin-macios-pr-builder/tests/xharness/tmp-test-dir/monotouch-test75/monotouch-test.csproj]
15:58:03.9074240 /Users/builder/jenkins/workspace/xamarin-macios-pr-builder/tests/monotouch-test/ObjCRuntime/RegistrarTest.cs(1080): error MT4102: The registrar found an invalid type `System.DateTime` in signature for method `MonoTouchFixtures.ObjCRuntime.RegistrarTest/InvalidDateTime.Foo`. Use `Foundation.NSDate` instead. [/Users/builder/jenkins/workspace/xamarin-macios-pr-builder/tests/xharness/tmp-test-dir/monotouch-test75/monotouch-test.csproj]
````
Should I give up on a test here? or someone have another idea?

Reply:
Yes, this will be hard to do with a unit test, because the failure will either show up at build time or during app startup.

I think it's fine to skip automating this test for now, and add an issue about it so that we don't forget. It'll be a much bigger change to add support for testing this scenario.

Reply:
I filed: https://github.com/xamarin/xamarin-macios/issues/8782